### ISSUE TITLE
Admin can view team without billing enabled and not know

### ISSUE THREAD

**Issue by joepavitt** [flowforge/flowforge#649]
### Current Behavior

Seems we have backtracked some changes which now allows a user to bypass Stripe. I am able to create a team, and project, without ever putting in billing information. Although, anytime I try to access the project, I get a 404, additionally, I cannot delete the team because of the active project (which I cannot delete).

If I try to insert billing information via the "Team Settings" option, the link to the "Stripe Customer Portal" returns a `500`:

```json
{
    "statusCode": 500,
    "error": "Internal Server Error",
    "message":" Cannot read properties of null (reading 'customer')"
}
```

### Expected Behavior

I should be prompted to enter billing information if none is present when creating a project.

### Steps To Reproduce

On an instance of FF with billing enabled:
- Create new team
- Click "Proceed to Stripe"
- Select "Back" option in the top-left of the Stripe UI.

You now have a team created, without billing information.

- Click "Create Project"
- You are prompted to input the relevant project information here. This should have been redirected to the billing settings page.

### Environment

- FlowForge version: 0.5.0
- Node.js version:
- npm version:
- Platform/OS: Mac
- Browser: Chrome

**Comment by joepavitt** [comment]
After a chat with @knolleary - seems this is due to a situation whereby admin users are given access to teams, independent of whether billing has been setup or not (in order to remove invalid teams/projects)

**Comment by knolleary** [comment]
I've updated the title to make it reflect our findings here.

**Comment by sammachin** [comment]
This is fixed in 0.10 with the changes to admin RBAC #940
